test/regression/aarch64/instructions/sme.cc

```diff
@@ -7,8 +7,52 @@ namespace {
 
 using InstSme = AArch64RegressionTest;
 
-#if SIMENG_LLVM_VERSION >= 14
```

Comment:
Are we not able to keep this check in? I assumed this just concerned no SME and SME support as opposed to SME and SME2 support. May have misremembered the LLVM versioning though

Reply:
Yeah you're correct. Will re-add